tests/regressiontests/test-simple.m4

```diff
@@ -12,7 +13,11 @@
 
 # Now we take the parsed data and assign them no nice-looking variable names,
 # sometimes after a basic validation
-echo "OPT_S=$_arg_prefix,POS_S=$_arg_pos_arg,LA=$_arg_la,"
+if [ "$_arg_print_optionals" = on ]; then
+    echo "OPT_S=$_arg_prefix,POS_S=$_arg_pos_arg,LA=$_arg_la,_supplied_arg_prefix=${_supplied_arg_prefix-x},_supplied_arg_print_optionals=${_supplied_arg_print_optionals-x},_supplied_arg_la=${_supplied_arg_la-x}"
```

Comment:
Here, for testing purposes, you can decide to drop echoing argument values in this branch of `if`, and focus only on printing the line with supplied arguments.
In the test itself, you can call the script with `--print-optionals` and with other args (or not), and then just check that the expected output matches.